editoast/openapi.yaml

```diff
@@ -3083,6 +3084,18 @@ components:
           type: string
         ch:
           type: string
+        track_sections:
+          type: array
+          items:
+            type: object
+            required:
+              - track
+              - position
+            properties:
+              track:
+                type: string
+              position:
```

Comment:
Are you sure it is position and not offset here ?

Reply:
Yes it's offset in the pathfinding request but here it's called position 🤷

Reply:
I could rename it but that would be done in in pgsql so I'd rather leave it that way 😅